api/proto/attest/attest.proto

```diff
@@ -0,0 +1,49 @@
+// Copyright(c) 2020 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+option go_package  = "github.com/lf-edge/eve/api/go/certs";
+option java_package = "com.zededa.cloud.uservice.proto";
+
+// ZAttestNonceReq is request payload for POST /api/v1/edgeDevice/attestNonce
+// or /api/v2/edgeDevice/attestNonce
+// ZAttestNonceResp is response payload for POST /api/v1/edgeDevice/attestNonce
+// or /api/v2/edgeDevice/attestNonce
+// The message is assumed to be protected by a TLS session bound to the
+// device certificate for v1.
+// The message is assumed to be protected by signing envelope for v2.
+
+message ZAttestNonceReq {
+  string deviceUuid = 1;  //UUID assigned by the controller during register
+}
+message ZAttestNonceResp {
+  string nonce = 1;      //nonce to use in quote generation
```

Comment:
bytes?

Reply:
> Hey @nordmark -- I have two requests to update the documentation, but most of all -- I'd rather not include Uuid field for now (this creates precedent in our API that we haven't had before and I'd like to be really convinced that there's a good use case for it).

@rvs, please check now. Removed UUID and updated API.md